en/frames_multicopter/dji_f450_cuav_5plus.md

```diff
@@ -0,0 +1,212 @@
+# DJI F450 + CUAV V5+ Build
+
+This topic provides full instructions for building the kit and configuring PX4 using *QGroundControl*.
+
+Key information
+
+- **Frame:** DJI F450
+- **Flight controller:** [CUAV V5+](../flight_controller/cuav_v5_plus.md)
+- **Assembly time (approx.):** 90 minutes (45 minutes for frame, 45 minutes autopilot installation/configuration)
+
+![Finished setup 1](../../assets/airframes/multicopter/dji_f450_cuav_5plus/f450_cuav5_plus_complete.jpg)
+
+## Bill of materials
+
+The components needed for this build are:
```

Comment:
@tony3dr
- Is all this stuff what comes in the kit from CUAV? Can separate by kit?
- This does not mention the CUAV v5+